### DISCUSSION TITLE
Workaround for displaying e.g. convention programm

### DISCUSSION THREAD

**Discussion by ux-git** [sircharlo/meeting-media-manager#2248] [Q&A] [answered]
We often show a convention/assembly programs before an event. Currently I can't come up with a solution how to demonstrate a program file, which is not and image/video is, to see the text with program points.

Is there any way to demonstrate it, other than using JW.ORG beta feature? The problem with it is that I need to open the JW.org each time and search for the program, but we need to show the program immediately instead of publishers seeing how we search the program, open it, zoom closer, etc.

**Comment by ux-git** [comment]
Maybe there's some hack like enabling printed Editions import and then import PDF somehow or something like that. If not, what do you think about adding a feature to cover this use case?

**Reply by e-Feitosa** [reply, marked as answer]
Hi, you can download the file in PDF format and import it into the M3, it will extract the pages as an image to be displayed.
🔹In the main panel, select the meeting date 📅
🔹Click on custom 
🔹Select the PDF file downloaded on your PC 📂
🔹Click open and that's it, follow the final steps and complete the import by clicking save 💾.

**Comment by mtdvlpr** [comment]
@ux-git, like @e-Feitosa said, we already cover this use case. PDF's are automagically converted to images when added to M3.
